app/scripts/metamask-controller.js

```diff
@@ -4742,7 +4757,8 @@ export default class MetamaskController extends EventEmitter {
     }
   };
 
-  async securityProviderRequest(requestData, methodName) {
+  ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-mmi,build-beta)
```

Comment:
This looks better 👍 
But now you need to code fence the [createRPCMethodTrackingMiddleware](https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/lib/createRPCMethodTrackingMiddleware.js#L116C25-L116C58) and it's usage of `securityProviderRequest`

Reply:
code fencing[createRPCMethodTrackingMiddleware.js]( https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/lib/createRPCMethodTrackingMiddleware.js#L116C25-L116C58) is breaking jsdoc because code fencing is not working for jsdoc

I have added a check if the method securityProviderRequest exists before calling it.